widlparser/productions.py

```diff
@@ -1062,6 +1081,14 @@ def _define_markup(self, generator: MarkupGenerator) -> Production:
 			generator.add_text(self._close_type)
 			generator.add_text(self.null)
 			return self
+		if (self.async_iterable):
+			self.async_iterable[0].define_markup(generator)
+			self.async_iterable[1].define_markup(generator)
+			generator.add_text(self._open_type)
+			self.type.define_markup(generator)
```

Comment:
```suggestion
			generator.add_type(self.type)
```

Reply:
Done. I had copied this from the `self.record` branch.